packages/ui/src/stories/atoms/Textarea.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import Textarea from "../../components/atoms/Textarea"
+
+const meta = {
+    title: "Atoms/Textarea",
+    component: Textarea,
+    parameters: {
+        layout: "centered",
+    },
+    tags: ["autodocs"],
+    argTypes: {
+        label: { control: 'text' },
```

Comment:
Is label visible? 
If it is not, is this control necessary?

Reply:
Is true, its not visible, will fix it